web/app/js/components/EdgesTable.jsx

```diff
@@ -1,113 +1,118 @@
-          return (
-            <PrefixedLink to={`/namespaces/${namespace}/${type}s/${d.src.name}`}>
-              {d.src.name}
-            </PrefixedLink>
-          );
-        } else {
-          return d.src.name;
-        }
-      },
-      sorter: d => d.src.name + d.dst.name
+      title: " ",
+      dataIndex: "direction",
+      render: d => directionColumn(d.direction),
     },
     {
-      title: "Destination",
-      dataIndex: "destination",
+      title: "Name",
+      dataIndex: "name",
       isNumeric: false,
-      filter: d => d.dst.name,
+      filter: d => d.name,
       render: d => {
-        // check that the destination is a k8s resource with a name we can link to
-        if (namespace && type && d.dst && d.dst.name) {
+        // check that the resource is a k8s resource with a name we can link to
+        if (namespace && type && d.name) {
           return (
-            <PrefixedLink to={`/namespaces/${namespace}/${type}s/${d.dst.name}`}>
-              {d.dst.name}
+            <PrefixedLink to={`/namespaces/${namespace}/${type}s/${d.name}`}>
+              {d.name}
             </PrefixedLink>
           );
         } else {
-          return d.dst.name;
+          return d.name;
         }
       },
-      sorter: d => d.dst.name + d.src.name
+      sorter: d => d.name
     },
     {
-      title: "Client",
-      dataIndex: "client",
+      title: "Identity",
+      dataIndex: "identity",
       isNumeric: false,
-      filter: d => d.clientId,
-      render: d => d.clientId.split('.')[0] + '.' + d.clientId.split('.')[1],
-      sorter: d => d.clientId
+      filter: d => d.identity,
+      render: d => d.identity ? d.identity.split('.')[0] + '.' + d.identity.split('.')[1] : null,
+      sorter: d => d.identity
     },
     {
-      title: "Server",
-      dataIndex: "server",
-      isNumeric: false,
-      filter: d => d.serverId,
-      render: d => d.serverId.split('.')[0] + '.' + d.serverId.split('.')[1],
-      sorter: d => d.serverId
-    },
-    {
-      title: "Message",
+      title: "Secured",
       dataIndex: "message",
       isNumeric: false,
-      filter: d => d.noIdentityMsg,
-      render: d => d.noIdentityMsg,
-      sorter: d => d.noIdentityMsg
+      render: d => {
+        if (d.noIdentityMsg === "") {
+          return <CheckCircleOutline className={classes.secure} />;
+        } else {
+          return (
+            <Tooltip title={d.noIdentityMsg}>
+              <WarningIcon className={classes.warning} />
+            </Tooltip>
+          );
+        }
+      }
     }
   ];
 };
 
-const tooltipText = `Edges show the source, destination name and identity
-  for proxied connections. If no identity is known, a message is displayed.`;
+const generateEdgesTableTitle = edges => {
+  let title = "Edges";
+  let identity = "";
+  if (edges.length > 0) {
+    edges[0].direction === "INBOUND" ? identity = edges[0].serverId : identity = edges[0].clientId;
```

Comment:
we don't use identity outside the if, so this would be simpler to not declare it on line 75, and declare it here as 
```
let identity = edges[0].direction === "INBOUND" ? edges[0].serverId : edges[0].clientId;
```